backend/src/database/migrations/20240206220107_create_db_log_table.js

```diff
@@ -0,0 +1,18 @@
+exports.up = function (knex) {
+  return knex.schema.withSchema("config").createTable("db_logs", function (table) {
+    table
+      .increments("id")
+      .primary()
+      .notNullable()
+      .defaultTo(knex.raw("nextval('config.id_seq'::regclass)"));
+    table.text("api_method");

```

Comment:
although probably not a big deal defining types closer to what they should be is always a good thing as appose to just defining them as text https://www.postgresql.org/docs/current/datatype-character.html